source/_components/switch.ads.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: "ADS Switch"
+description: "Instructions on how to set up ADS switches within Home Assistant."
+date: 2017-10-25 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Switch
+ha_release: 0.56
+ha_iot_class: "Local Push"
+---
+
+An ADS switch accesses a boolean variable on the connected ADS device. The
+variable is identified by its name.
+
+To use your ADS device, you first have to set up your [ADS
+hub](/components/ads/) and then add the following to your `configuration.yaml`
+file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: ads
+    adsvar: .global_bool
+```
+
+Configuration variables:
+
+- **adsvar** (*Required*): The name of the variable which you want to access on
```

Comment:
Please use the configuration tags for listing configuration variables. For more information:
https://home-assistant.io/developers/documentation/create_page/#configuration